integration/libs/git/git_fetch_test.go

```diff
@@ -39,19 +38,18 @@ func assertSparseGitInfo(t *testing.T, expectedRoot string, info git.RepositoryI
 	assert.Equal(t, expectedRoot, info.WorktreeRoot)
 }
 
+func ensureWorkspacePrefix(root string) string {
+	// The fixture helper doesn't include /Workspace, so include it here.
```

Comment:
Why not fix the fixture helper instead to generate right path?

Reply:
I tried this but there are assumptions elsewhere on the prefix not being included.

I'd like to address in this a separate PR and normalize on having the `/Workspace` prefix everywhere.